launchers/VirSat Core Application (demo).launch

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Most likely the launcher should be renamed as well. We should not call it "Beta" here.

Reply:
It's called "IgnoreBeta" now... Or do you mean you don't like "Beta" in the development environment. I think it's ok because it removes the "Beta" warnings from the UI. What do you think?

Reply:
Its just a minor detail. it is just important, that we know what the individual launchers stand for.